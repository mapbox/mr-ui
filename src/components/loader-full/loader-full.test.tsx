import React from 'react';
import { render } from '@testing-library/react';
import LoaderFull from './loader-full';

describe('LoaderFull', () => {
  describe('basic', () => {
    test('renders', () => {
      const { baseElement } = render(<LoaderFull />)
      expect(baseElement).toMatchSnapshot();
    });
  });

  describe('opaque', () => {
    test('renders', () => {
      const { baseElement } = render(<LoaderFull opaque={true} />)
      expect(baseElement).toMatchSnapshot();
    });
  });
});
