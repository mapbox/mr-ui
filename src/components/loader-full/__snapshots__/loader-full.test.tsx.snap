// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderFull basic renders 1`] = `
<body>
  <div />
  <div>
    <div
      class="fixed top left right bottom flex flex--center-main flex--center-cross bg-darken50"
      data-testid="loader"
      style="z-index: 9999;"
    >
      <div
        class="loading loading--dark"
      />
    </div>
  </div>
</body>
`;

exports[`LoaderFull opaque renders 1`] = `
<body>
  <div />
  <div>
    <div
      class="fixed top left right bottom flex flex--center-main flex--center-cross bg-white"
      data-testid="loader"
      style="z-index: 9999;"
    >
      <div
        class="loading"
      />
    </div>
  </div>
</body>
`;
